ee/se/modules/380-metallb/hooks/discovery_l2_load_balancers.go

```diff
@@ -181,7 +157,7 @@ func handleLoadBalancers(input *go_hook.HookInput) error {
 				ServiceName:              service.Name,
 				ServiceNamespace:         service.Namespace,
 				PreferredNode:            nodes[nodeIndex].Name,
-				LoadBalancerClass:        service.LoadBalancerClass,
+				LoadBalancerClass:        l2lb.Name,
```

Comment:
```suggestion
```
Don't need the LoadBalancerClass here. We hardcoded it in templates/l2lbservice/l2lbservice.yaml.